python/README.md

```diff
@@ -18,3 +18,20 @@ In order to obtain relevant modules, you should install them from PyPI directly,
 ```
 pip install geoarrow-rust-core
 ```
+
+## Development
```

Comment:
I'd prefer to move this into `DEVELOP.md` as normal end-users don't need to consider any of this.

Reply:
Will do..

Reply:
Done